src/translib/transformer/xfmr_bgp_nbr.go

```diff
@@ -647,6 +647,16 @@ func fill_nbr_state_cmn_info (nbr_key *_xfmr_bgp_nbr_state_key, frrNbrDataValue
             nbrState.CapabilityExtendedNexthop = &_capabilityExtendedNexthop
         }
 
+        if value, ok := cfgDbEntry["bfd"] ; ok {
```

Comment:
In YANG we added some new attrs to NBR,  config we are getting from DB, the some attrs are state which we are filling in State subtree, support for these new attrs added in subtree?

Reply:
Few state fields need to be added, will add them in the next commit.